app/main.py

```diff
@@ -1 +1,30 @@
-# write your code here
+import os
+import shutil
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
```

Comment:
Code Style: The variable name `parts` is not descriptive enough. Consider renaming it to something more meaningful, like `command_parts`.